core/loadpoint_vehicle.go

```diff
@@ -151,9 +152,7 @@ func (lp *Loadpoint) setActiveVehicle(v api.Vehicle) {
 		lp.progress.Reset()
 	} else {
 		lp.socEstimator = nil
-		lp.publish(keys.VehicleSoc, 0)
-		lp.publish(keys.VehicleName, "")
-		lp.publish(keys.VehicleOdometer, 0.0)
```

Comment:
remaining attributes are already published in  https://github.com/evcc-io/evcc/pull/21720/files#diff-327717b3100908ca99e06df09e2e631ed2925f9a995d5e7ef3dcd0a036af6948R160